### DISCUSSION TITLE
Using Blazorise SpinKit in Blazor Web Assembly

### DISCUSSION THREAD

**Discussion by GevorDanielyan** [Megabit/Blazorise#3687] [Q&A]
Hello everyone.

I'd like to know how I can use SpinKit in index.html in my WASM project instead of  `<div id="app">Loading...</div>` on first running application?

Thanks in advance.

**Comment by stsrki** [comment]
Hi @GevorDanielyan 

Using any Blazor component instead of `<div id="app">Loading...</div>` is not possible. The reason is that `<div id="app">Loading...</div>` is the placeholder where Blazor will be rendered once it is initialized. That's why you would have to manually define HTML elements instead. Like we did in our demo https://github.com/Megabit/Blazorise/blob/master/Demos/Blazorise.Demo.Bootstrap/wwwroot/index.html#L30

**Comment by GevorDanielyan** [comment]
Hi @stsrki .

Thank you for your response. I will try. Thanks.
